### DISCUSSION TITLE
Is there a native way to have table rows in alternating colors?

### DISCUSSION THREAD

**Discussion by aborruso** [squidfunk/mkdocs-material#2800] [Q&A] [answered]
Hi,
first of all: thank you, this is a great project.

Probably the answer to my question is no. I think it is a very interesting feature and I also propose it as a feature request.

Thank you

**Comment by aborruso** [comment, marked as answer]
If each row alternately had the even and odd class, it could be a good starting point. I don't open pull request, because I am unable to write this code change

**Reply by squidfunk** [reply]
This should be achievable with custom CSS in a few minutes, e.g. [like this](https://www.w3schools.com/css/tryit.asp?filename=trycss_table_striped).

**Reply by aborruso** [reply]
Brilliant, thank you
